### ISSUE TITLE
 how to change the text of the select ionic cancel button

### ISSUE THREAD

**Issue by Luancarlos** [ngx-formly/ngx-formly#1963]
**Question**

A clear and concise description of what the question is.

**Comment by Devqon** [comment]
Looking in the code this isn't supported (yet). You can do either:
- Fork the repo, create a pull request with the change
- Create a custom template for the ion-select which contains the

You need to include the `cancelText` (and if you're already on it, the `okText`) in the template:

import { Component } from '@angular/core';
import { FieldType } from '@ngx-formly/core';

```
    <ion-select
      [cancelText]="to.cancelText || 'Cancel'"
      [okText]="to.okText || 'OK'"
      [formControl]="formControl"
      [ionFormlyAttributes]="field"
      [multiple]="to.multiple"
      [interface]="to.interface">
      <ion-select-option *ngFor="let option of to.options | formlySelectOptions:field | async" [value]="option.value">
        {{ option.label }}
      </ion-select-option>
    </ion-select>
```

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v5.5.7](https://github.com/ngx-formly/ngx-formly/releases/tag/v5.5.7) release.

@Devqon thanks for the hint 👍!!

**Comment by Luancarlos** [comment]
thank you very much, you are very good
